plugins/fileservices/fileservices.cpp

```diff
@@ -635,7 +635,8 @@ static void blockUntilComplete(const char * label, IClientFileSpray &server, ICo
         if (wu.get()) { // if updatable (e.g. not hthor with no agent context)
             aborting = wu->aborting();
             StringBuffer wuScope, ElapsedLabel, RemainingLabel;
-            wuScope.appendf("%s-%s", label, dfuwu.getID());
+            StringBuffer labelbuf(label);
+            wuScope.appendf("dfu:%s:%s", labelbuf.toLowerCase().str(), dfuwu.getID());
```

Comment:
I think this is reasonable.  Being consistent both dfu and the copy operation should also be valid scopes - especially if values are ever aggregated into them.

I started introducing a similar change for sections in a compile (see https://github.com/hpcc-systems/HPCC-Platform/pull/17483), which looks like it has stalled.  I think it would be useful to have a couple of general purpose items.  Maybe SSTcategory which could be used for your two examples above, and SSTstage which I was introducing in mine - or SSTcategory could be used for both.

Reply:
blockUntilComplete is called for the following files services activity: Spray, Despray, Copy, Replicate, WaitDFUWorkunit, MonitorLogicalFileName, MonitorFile and RemotePull.  I have used "dfu" as the parent scope for all of these.  And so "copy" would a child scope of "dfu".

I see the benefit of prefixing the operation name with a an identifier like ">"  It would make easier StatsScopeId to parse from a scope string.

We should discuss SSTcategory/SSTstage on teams.